File: test_data/code/java/1385-find-the-distance-value-between-two-arrays.java
class Solution {
    public int findTheDistanceValue(int[] arr1, int[] arr2, int d) {
        int count = 0;
        for (int i : arr1) {
            boolean found = false;
            for (int j : arr2) {
                if (Math.abs(i - j) <= d) {
                    found = true;
                    break;
                }
            }
            if (!found) {
                count++;
            }
        }
        return count;
    }
}

File: test_data/code/python/1385-find-the-distance-value-between-two-arrays.py
Python
class Solution:
    def findTheDistanceValue(self, arr1: List[int], arr2: List[int], d: int) -> int:
        count = 0
        for num in arr1:
            flag = True
            for num2 in arr2:
                if abs(num - num2) <= d:
                    flag = False
                    break
            if not flag:
                continue
            count += 1
        return count